### PR TITLE
update releasenote1.19 and some other docs

### PR DESCRIPTION
**Pull request by wangsimo0**
update releasenote1.19 and some other docs

### CODE REVIEW

File: administration/Blacklist.md [hunk 1]
```diff
@@ -0,0 +1,107 @@
+# Blacklist Management
+
+In some cases, administrators need to disable certain patterns of SQL to avoid SQL from triggering cluster crashes or unexpected high concurrent queries.
+
+StarRocks allows users to add, view, and delete SQL blacklists.
+
+## Syntax
+
+EnabeSQL blacklisting via `enable_sql_blacklist`. The default is False (off).
```
Comment (chaoyli) [inline administration/Blacklist.md:9]: EnableSQL

File: benchmarking/SSB_Benchmarking.md [hunk 2]
```diff
@@ -0,0 +1,446 @@
...
+
+### Data import
+
+We import the data using stream load
+
+```shell
+bin/stream_load.sh data_dir
+```
+
+`data\_dir` is the data directory.
```
Comment (chaoyli) [inline benchmarking/SSB_Benchmarking.md:428]: data_dir

File: administration/Load_balance.md [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+### 1. Install relevant dependencies
+
+~~~shell
+yum install -y gnutls perl-DBD-MySQL perl-DBI perl-devel
+~~~
+
+### 2. Download the installation package
+
+~~~shell
+wget https://github.com/sysown/proxysql/releases/download/v2.0.14/proxysql-2.0.14-1-centos7.x86\_64.rpm
```
Comment (chaoyli) [inline administration/Load_balance.md:34]: x86_64

File: administration/Profiling.md [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+
+* Data can be distinguished between hot and cold
+* Be able to leverage StarRocks tiered storage (SSD + SATA)
+* Faster to delete data by partition
+
+* HASH bucket: Divides data into different buckets according to the hash value.
+
+* It is recommended to use a column with a large degree of differentiation for bucketing to avoid data skewing
+* To facilitate data recovery, it is recommended to keep  the size of each bucket around 10GB. Please consider the number of buckets when creating a table or adding a partition .
+* Random bucketing is not recommended, please specify a clear hash for bucketing.
```
Comment (chaoyli) [inline administration/Profiling.md:126]: Random bucketing is already removed in code paths.

File: administration/Blacklist.md (hunk 1 above)
Comment (wangsimo0) [reply administration/Blacklist.md]: done

File: administration/Load_balance.md (hunk 3 above)
Comment (wangsimo0) [reply administration/Load_balance.md]: done

File: administration/Profiling.md (hunk 4 above)
Comment (wangsimo0) [reply administration/Profiling.md]: done

File: benchmarking/SSB_Benchmarking.md (hunk 2 above)
Comment (wangsimo0) [reply benchmarking/SSB_Benchmarking.md]: done

**Review by chaoyli — APPROVED** [review]

**Pull request merged**